base_user_annotation4.nit:16,2--19: Warning: an annotation `foo` is already declared in module `base_user_annotation`
